config/samples/ccruntime/default/kustomization.yaml

```diff
@@ -27,13 +27,13 @@ patches:
         pulltype: ""
       - name: "kata-qemu-tdx"
         snapshotter: "nydus"
-        pulltype: ""
+        pulltype: "guest-pull"
```

Comment:
Hi @fidencio !

We don't test TDX/SEV/SNP with CRI-O so I left this empty on purpose. But it was a mistake, we better have it set to guest-pull anyway, hence, thanks for the fix!

BTW, do you plan to test TDX + CRI-O? There is this https://github.com/confidential-containers/operator/issues/409 and I asked on last CI meeting  if platform folks would like to test with CRI-O (I didn't get a 'yes' nor 'no'...)

Reply:
Right now we're struggling to have machines available to test.
This, however, helps me when doing tests for Red Hat related stuff. ;-)

Reply:
Got it. yeah, it would be trick to use same machine to containerd and cri-o and keep switching the configurations....